parlai/tasks/eli5/README.md

```diff
@@ -0,0 +1,143 @@
+# ELI5
+**Adapted from Facebook Research's ELI5 found at https://github.com/facebookresearch/ELI5/**
```

Comment:
Thanks for including a README. In general, I think it would be better if a few parts of this were rewritten from the ParlAI standpoint, i.e. as a platform that provides these scripts as a way to download and use ELI5 _within ParlAI_; as such, a lot of the talking points here should be left in the regular ELI5 README and perhaps just  _referenced_ from this README. I won't correct everything I see but will make comments here and there and I think you'll get the general idea of what we want this README to look like

Reply:
Agreed

Reply:
If you really want, you can like, do this:

````markdown
# ELI5

This is the ParlAI adaptation of the ELI5 dataset. It contains a lot of moving parts, most of which are adapted from the [original repo](https://github.com/facebookresearch/ELI5/).

The instructions for using it are as follows:
1. Step1
2. Step2

Below this point, you will find the **original README** taken from the ELI5 repo. It is left here only to provide additional context, and does NOT reflect installation instructions in ParlAI.

-----
Continued....
```

Reply:
I changed the header section a little bit, but the rest is pretty much just the steps to get the dataset. Also, I left the citation section in there. Not sure if I should leave that or not